scripts/artifacts-building/containers/build-process.sh

```diff
@@ -131,10 +131,10 @@ if ! python_artifacts_available; then
     # which no longer exists.
     # We implement the override here to cater for PR tests
     # only.
-    eol_repo_list="cinder glance heat horizon keystone magnum_dashboard"
+    eol_repo_list="cinder glance heat ironic horizon keystone magnum_dashboard"
```

Comment:
The list appears to be alphabetical, so this is in the wrong place.

Reply:
Ah, good catch - fixing.